redis/client.py

```diff
@@ -1530,6 +1530,8 @@ def handle_message(self, response, ignore_subscribe_messages=False):
         with a message handler, the handler is invoked instead of a parsed
         message being returned.
         """
+        if response is None:
```

Comment:
Looks like this made it into this PR as well - but shouldn't have. Can you reseed, without this?